_rules/text-node-contrast-afw4f7.md

```diff
@@ -146,21 +147,31 @@ Text that does not express anything in human language.
 
 #### Passed Example 9
 
-Text that is in a shadow DOM tree with sufficient contrast.
+This text is in a shadow DOM tree and has sufficient contrast.
 
 ```html
 <p style="color: #333; background: #fff;" id="p"></p>
 <script>
-  const shadowRoot = document.getElementById('p').attachShadow({ mode: 'open' })
-  shadowRoot.textContent = 'Some text in English'
+	const shadowRoot = document.getElementById('p').attachShadow({ mode: 'open' })
+	shadowRoot.textContent = 'Some text in English'
 </script>
 ```
 
+#### Passed Example 10
+
+This black [text node](https://dom.spec.whatwg.org/#text) is in Japanese, showing that this rule applies to any human language. (The prase means "Even monkeys fall from trees.").
```

Comment:
I'm still understanding the sentence (and example) as "the rule applies to the primary language of the page, whatever it is" rather than "the rule applies to every text on the page, no matter what language it's in, even if it is different from the primary language of the page".

Imho, the example to show that would be (keeping the same idea) a page in English listing Japanese sayings (and their translations) (only one being actually needed in the example) and thus including some text in Japanese. If the English text has good contrast but the Japanese doesn't, then the failure of the rule shows that it is concerned about any text on the page, not only text in the primary language.
(if both Japanese and English text have good contrast, then it is still possible to believe that the rule does not care about the Japanese part)